_config.yml

```diff
@@ -6,8 +6,8 @@ website: "https://wet-boew.github.io/GCWeb/" # URL of your public facing website
 global:
   lang: fr
   feedbackPath:
-    en: "/sites/feedback/ajax/page-feedback-en.html"
-    fr: "/sites/feedback/ajax/page-feedback-fr.html"
+    en: "/dist/GCWeb/assets/feedback/page-feedback-en.html"
+    fr: "/dist/GCWeb/assets/feedback/page-feedback-fr.html"
```

Comment:
We would need the absolute URL here, because it is broken in gh-page. 

Like: https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/feedback/page-feedback-en.html